components/google_sheets/sources/new-row-added/new-row-added.mjs

```diff
@@ -1,172 +1,25 @@
-import common from "../common.mjs";
+import httpBase from "../common/http-based/sheet.mjs";
+import newRowAdded from "../common/new-row-added.mjs";
 
 export default {
-  ...common,
+  ...httpBase,
+  ...newRowAdded,
   key: "google_sheets-new-row-added",
   name: "New Row Added (Instant)",
-  description:
-    "Emit new events each time a row or rows are added to the bottom of a spreadsheet",
-  version: "0.0.24",
+  description: "Emit new events each time a row or rows are added to the bottom of a spreadsheet. To use this source with a spreadsheet in a [Shared Drive](https://support.google.com/a/users/answer/9310351), use the **New Row Added (Shared Drive, Instant)** source instead.",
+  version: "0.0.25",
   dedupe: "unique",
   type: "source",
+  hooks: {
+    ...httpBase.hooks,
+    ...newRowAdded.hooks,
+  },
```

Comment:
Since `newRowAdded` doesn't have any hooks, and the hooks from `httpBase` will be included anyway, this hooks section can be eliminated.
```suggestion
```